features/keychain/module/keychain.js

```diff
@@ -62,8 +59,6 @@ export class Keychain extends ExodusModule {
   }
 
   #getPrivateHDKey = ({ seedId, keyId }) => {
-    throwIfInvalidKeyIdentifier(keyId)
```

Comment:
There should be validation here, since  this is the function that is used by all other functions.
Moving the validation to just `exportKey` isn't great, I would prefer  `new KeyIdentifier(keyId)` in the api then

Reply:
`new KeyIdentifier` performs all the validation needed

Reply:
Yes, but it has been moved  to `exportKey`, meaning it doesn't validation any functions exported by `keychain.sodium`, `keychain.ed25519`, `keychain.secp256k1`,

Reply:
aaah, you're right bea945e0a1725206f167b5c21d5a6fbd91526c00